WSDOTAgent/GirderScheduleChapterBuilder.cpp

```diff
@@ -683,6 +684,34 @@ rptChapter* CGirderScheduleChapterBuilder::Build(const std::shared_ptr<const WBF
       }
    }
 
+   Float64 tft = pIGirder->GetTopFlangeThickening(segmentKey);
+   if (!IsZero(tft))
```

Comment:
The test should be 
~~~
if(pIGirder->CanTopFlangeBeLongitudinallyThickened(segmentKey) && !IsZero(tft))
~~~

Top flange thickening is not guaranteed to be zero for beams that don't support top flange thickening.